cmd/world/root/root.go

```diff
@@ -44,6 +52,63 @@ func init() {
 	logger.AddLogFlag(doctorCmd)
 }
 
+func checkLatestVersion() error {
+	url := "https://api.github.com/repos/Argus-Labs/world-cli/releases/latest"
+
+	// Create a request with the context
+	req, err := http.NewRequestWithContext(context.Background(), http.MethodGet, url, nil)
+	if err != nil {
+		return fmt.Errorf("error creating request: %s", err.Error())
```

Comment:
use eris.Wrap(err, "error creating request") so you also get stack trace, this applies for other errors too.

Reply:
updated